Source/ACE/Entity/Landblock.cs

```diff
@@ -477,6 +477,13 @@ public void UseTime()
                     QueuedGameAction action = player.ActionQueuePop();
                     if (action != null)
                         HandleGameAction(action, player);
+                    else

```

Comment:
Is there a reason we'd only process the Examination Queue when there wasn't a pending action?  I don't get the point of putting this in an else block.